src/client/__tests__/testKits/apis/githubApi.ts

```diff
@@ -0,0 +1,54 @@
+import GitHub from 'github-api';
+import { anyString, instance, mock, when } from 'ts-mockito';
+import { IApiTestClass } from './IApiTestClass';
+
+export class GitHubApiTestKit implements IApiTestClass<GitHub> {
```

Comment:
Rename this file to `GitHubApiTestKit.ts`, same as the class name.

Reply:
Done